packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -194,7 +194,7 @@ class RequestWrapper {
     } catch (err) {
       if (this.pluginCallbacks.fetchDidFail) {
         for (let callback of this.pluginCallbacks.fetchDidFail) {
-          callback({request});
+          callback({request: clonedRequest});
```

Comment:
You'll run into the same problem here if there are multiple `fetchDidFail` callbacks and the first one consumes the `clonedRequest` body. You can either change this to

```js
callback({request: clonedRequest.clone()});
```

which has the drawback of always creating 1 extra clone beyond what's needed, or alternatively, do something where you check the length of `this.pluginCallbacks.fetchDidFail`, create an array containing enough extra clones to match that length, and then pop clones off that array so that you always use the exact number you need.